backends-velox/src/test/scala/org/apache/gluten/execution/ScalarFunctionsValidateSuite.scala

```diff
@@ -41,6 +41,12 @@ class ScalarFunctionsValidateSuite extends FunctionsValidateTest {
     }
   }
 
+  test("Test getbit function") {
+    runQueryAndCompare("SELECT getbit(l_partkey, 0) from lineitem limit 1") {
+      checkGlutenOperatorMatch[ProjectExecTransformer]
```

Comment:
I think it's better to move this into the above test. You can rename that test name to cover `getbit`.

Reply:
Done!